README.md

```diff
@@ -50,6 +50,65 @@ The python notebooks are written in [Jupyter](http://jupyter.org/).
    5. Now we are able to view and edit the notebooks on the browser using the URL: http://localhost:8888/tree/mxnet-notebooks/python/outline.ipynb
 
 
+### Scala
+
+The scala notebooks are written in [Jupyter](http://jupyter.org/) using [Jupyter-Scala Kernel V0.3.x](https://github.com/alexarchambault/jupyter-scala).
+
+- **Run** We can run and modify these notebooks if both [mxnet scala package](http://mxnet.io/get_started/index.html#setup-and-installation), [jupyter](http://jupyter.org/) and Jupyter-Scala Kernel are installed. There are various options for jupyter scala kernel. You can choose whichever you like.
+
+  If you have a AWS account, here is an easier way to run the notebooks:
+
+  1.  Launch a g2.2xlarge or p2.2xlarge instance by using AMI `ami-fe217de9` on N. Virginia (us-east-1). This AMI is built by using  [this script](https://gist.github.com/mli/b64322f446b2043e3350ddcbfa5957be). Remember to open the TCP port 8888 in the security group.
+
+  2.  Once launch is succeed, setup the following variable with proper value
+
+    ```bash
+      export HOSTNAME=ec2-107-22-159-132.compute-1.amazonaws.com
```

Comment:
A user might simply copy/paste what you have here and realize it does not work.
- Instead of a real host name, I'd suggest changing this to export HOSTNAME=<ec2-instance-public-ip>
- export PERM=<path-to-pem-file>